<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SuperWindow</class>
 <widget class="QMainWindow" name="SuperWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_show">
    <property name="geometry">
     <rect>
      <x>508</x>
      <y>252</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>绘图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBotton_AddLine">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>38</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加线类图形</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBotton_AddCircle">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>157</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加圆形图形</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBotton_DeleteObject">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>240</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除所选部件</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_line_type">
    <property name="geometry">
     <rect>
      <x>361</x>
      <y>127</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>10</number>
    </property>
    <item>
     <property name="text">
      <string>DoubleInfiniteLine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SingleInfiniteLine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FiniteLine</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spin_line_x1">
    <property name="geometry">
     <rect>
      <x>363</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_line_y1">
    <property name="geometry">
     <rect>
      <x>472</x>
      <y>40</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_line_x2">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>88</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_line_y2">
    <property name="geometry">
     <rect>
      <x>473</x>
      <y>88</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_circle_y">
    <property name="geometry">
     <rect>
      <x>364</x>
      <y>160</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_circle_x">
    <property name="geometry">
     <rect>
      <x>251</x>
      <y>160</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_circle_r">
    <property name="geometry">
     <rect>
      <x>473</x>
      <y>160</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBotton_LoadFile">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>200</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>加载文件</string>
    </property>
   </widget>
   <widget class="QListView" name="logView">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>441</width>
      <height>591</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_addLine_type">
    <property name="geometry">
     <rect>
      <x>327</x>
      <y>130</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>类型</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_addLine_x1">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>42</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_addLine_y1">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>43</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_addLine_x2">
    <property name="geometry">
     <rect>
      <x>343</x>
      <y>91</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_addLine_y2">
    <property name="geometry">
     <rect>
      <x>451</x>
      <y>91</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_addCircle_x">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>162</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_addCircle_y">
    <property name="geometry">
     <rect>
      <x>346</x>
      <y>163</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_addCircle_r">
    <property name="geometry">
     <rect>
      <x>455</x>
      <y>162</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_num_of_intersects">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>386</x>
      <y>255</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_addCircle_x_2">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>255</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Num Of Intersects</string>
    </property>
   </widget>
   <widget class="QListView" name="objectView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>561</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_addCircle_x_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>620</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Objects</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_addCircle_x_4">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>620</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
